src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -586,22 +587,22 @@ bool TriangleCountExecutor::proceedOrNot(std::set<string> partitionSet,int parti
         }
     }
 
+    bool result;
     if (entryWithMinValue.first == partitionId) {
         int currentWeight = aggregateWeightMap[entryWithMinValue.first];
         currentWeight++;
         aggregateWeightMap[entryWithMinValue.first] = currentWeight;
         triangleCount_logger.log("###COMPOSITE### Aggregator Initiated : Partition ID: " + std::to_string(partitionId) + " Weight : " + std::to_string(currentWeight), "info");
-        return true;
+        result = true;
     } else {
```

Comment:
We could remove this else check once we initialize the variable to `false`.

Reply:
Removed the redundant else block

Reply:
Ok